__tests__/app.test.js

```diff
@@ -49,10 +50,51 @@ describe("GET categories", () => {
             .then(({
                 body
             }) => {
-                console.log(
-                    body.msg
-                )
                 expect(body.msg).toBe("Sorry we cant find that end point")
             })
     });
+
+})
+
+describe('GET Reviews by id', () => {
+    test('return the correct review', () => {
+        return request(app)
+            .get("/api/reviews/1")
+            .expect(200)
+            .then(({
+                body
+            }) => {
+                expect(body.review.review_id).toBe(1)
+                expect(body.review.title).toBe('Agricola')
+                expect(body.review.designer).toBe('Uwe Rosenberg')
+                expect(body.review.owner).toBe('mallionaire')
+                expect(body.review.review_img_url).toBe('https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png')
+                expect(body.review.review_body).toBe('Farmyard fun!')
+                expect(body.review.category).toBe('euro game')
+                expect(body.review.votes).toBe(1)
+
+            })
+    });
+    test('when passed a number that doesnt exsist return 404', () => {
+        return request(app)
+            .get("/api/reviews/1000")
+            .expect(404)
+            .then(({
+                res
+            }) => {
+                expect(res.text)
+                    .toBe("Sorry Review cant be found")
+            })
```

Comment:
Here you should be destructuring the `{ body }` key instead of res. This is because you sent the wrong response format in your custom error handler. These tests should check the `body.msg` as your other tests did once you've corrected the error handler.